README.md

```diff
@@ -175,10 +174,6 @@ SSR pages and API endpoints. It is currently not possible to create custom Netli
 
 ## Caveats
 
-### Preview Mode
```

Comment:
Let's add a sentence about "preview mode not yet being available locally for pages without revalidate or fallback" to the section "Preview Locally"?